### DISCUSSION TITLE
[Docs]: Update Waybar Section

### DISCUSSION THREAD

**Discussion by darkeddie** [JaKooLit/Hyprland-Dots#291] [Documentation]
### Description

Provide a place that all Styles and Configurations can be viewed, that is organised so that any changes can be made by re-uploading a screenshot

### Current Documentation

[Here](https://github.com/JaKooLit/Hyprland-Dots/wiki/Gallery#waybars)

### Proposed Documentation

[Here](https://github.com/darkeddie/Hyprland-Dots/wiki) 

### Additional Information

All that needs to be done is the screenshots copied over from my repo and the link to be updated with your username, I used the same folder structure in the repo, or I could do a pull request to add them

**Comment by JaKooLit** [comment]
😻 wow.... great effort..

I am not sure if can do a wiki push

try.. if you cant, let me know so I can clone your wiki and then Ill integrate and adjust / upload images

**Comment by darkeddie** [comment]
No I couldn't, I think you have to be owner to do it, but it's setup so if you put the images in your waybar folder of the screenshots repo and change my user to yours it should just work

**Comment by JaKooLit** [comment]
oh cool. no worries. Ill figure it out.. thanks alot

<img src="https://raw.githubusercontent.com/JaKooLit/Telegram-Animated-Emojis/main/People/Love%20You%20Gesture.webp" alt="Sparkles" width="38" height="38" />

**Comment by JaKooLit** [comment]
Check this out :)

https://github.com/JaKooLit/Hyprland-Dots/wiki/Gallery

**Comment by darkeddie** [comment]
Looks good, there is just a formatting error on the first line of "Waybar Styles".
But much easier to tell what each style looks like without changing them :)

**Reply by JaKooLit** [reply]
Thanks.. I have updated it
